Makefile

```diff
@@ -0,0 +1,29 @@
+# Common makefile commands & variables between projects
+include .make/common.mk
+
+# Common Golang makefile commands & variables between projects
+include .make/go.mk
+
+## Not defined? Use default repo name which is the application
+ifeq ($(REPO_NAME),)
+	REPO_NAME="go-paymail"
+endif
+
+## Not defined? Use default repo owner
+ifeq ($(REPO_OWNER),)
+	REPO_OWNER="tonicpow"
```

Comment:
Should we change it to bitcoin-sv?

Reply:
I believe this will be handled in https://bitcoinassociation.atlassian.net/browse/BUX-142

Reply:
I think that this task is to change other repos which are using go-paymail from tonicpow organization